specs/MultiProfile.md

```diff
@@ -328,6 +362,18 @@ interface ICoreWebView2Profile2 : ICoreWebView2Profile {
   /// See ICoreWebView2CookieManager.
   [propget] HRESULT CookieManager([out, retval] ICoreWebView2CookieManager** cookieManager);
 }
+
+[uuid(1c1ae2cc-d5c2-ffe3-d3e7-7857035d23b7), object, pointer_default(unique)]
+interface ICoreWebView2Profile3 : ICoreWebView2Profile2 {
+  /// All webviews on this profile will be closed, and the profile will be marked for deletion.
+  /// After the Delete() call completes, The render process of webviews on this profile will
+  /// asynchronously exit with the reason:`COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED`.
+  /// See 'COREWEBVIEW2_PROCESS_FAILED_REASON::COREWEBVIEW2_PROCESS_FAILED_REASON_PROFILE_DELETED'
+  /// for more details. The profile directory on disk will be actually deleted when the browser
+  /// process exits. Profile creation will fail if you create a new profile with the same name
+  /// as a profile that is being deleted.
```

Comment:
What happens if you delete a profile, and then somebody asks for the same profile to be created? Since we don't clean up old profiles until the browser process exits, the CreateCoreWebView2ControllerAsync cannot succeed.

Reply:
**What happens if you delete a profile, and then somebody asks for the same profile to be created**
The create will failure.

Reply:
How does the CreateCoreWebView2ControllerAsync fail? Does it produce a null Controller? Does it throw an exception?

Reply:
Yes, please document the manner in which we will fail including the HRESULT error code.

Reply:
The HRESULT will be kUnexpectedError, just like create profile failure with other reason.